.travis.yml

```diff
@@ -29,6 +31,11 @@ install:
       elif [ "$TRAVIS_PYTHON_VERSION" = "pypy3.5" ]; then
         pip install -U click h11 wsproto==0.13.*;
         pip install -U autoflake codecov isort pytest pytest-cov requests;
+      elif [ "$TRAVIS_PYTHON_VERSION" = "3.8" ]; then
+        pip install -U Cython;
+        pip install -U git+https://github.com/MagicStack/uvloop.git@1eb6727437430fe0500d7f044dd6e4432abd0901;
+        pip install -U click h11 httptools websockets==8.* wsproto==0.13.*;
+        pip install -U autoflake black codecov isort pytest pytest-cov requests;
```

Comment:
How come we need to install cython here?
I guess ideally we'd have test cases that just `skipif` a required dependency isn't installed.
Then we can just eg. omit uvloop, until they have a non-dev version that supports 3.8.

Reply:
It was experimental PR to see what it gets to get it working and it doesn't look satisfying.
It definitely a good idea to have a fallback to `asyncio` with probably a warning.

Reply:
We'd be okay on this *if* we'd resolved https://github.com/encode/uvicorn/pull/298, *and* if we made the uvloop install conditional on py<=3.7 for the time being.

Users *can* still run on 3.8, but only if they figure out that they need to use `--loop asyncio`, which isn't ideal.

Reply:
Oh actually we'd got a conditional install for uvloop already. Is there a way we can also add "only for 3.6 and 3.7" to setup.py, until we've got a fix from them?

Reply:
Ah okay, we should probably just sit tight on this... https://github.com/MagicStack/uvloop/issues/277#issuecomment-544308916